plugins/processors/dcos_metadata/README.md

```diff
@@ -31,9 +37,11 @@ This process adds the following tags to any metric with a container_id tag set:
  - `service_name` - the name of the service (mesos framework) which scheduled 
                     the task associated with this container
 
-Additionally, any task labels which are prefixed with `DCOS_METRICS_` are added
-to each metric as a tag. For example, the application configuration would have
-every metric associated with it decorated with a `FOO=bar` tag.
+Additionally, any task labels which are prefixed with strings included in the configurable whitelist of prefixes
```

Comment:
This does not mention `whitelist`?

Reply:
Mentioned [below](https://github.com/dcos/telegraf/pull/29/files#diff-03ee7142860155190bd1bb03aefbc955R43)